recipes/icu/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+if [ "$(uname)" == "Linux" ]; then
+    ./configure --prefix="$PREFIX"
+fi
+
+if [ "$(uname)" == "Darwin" ]; then
+    export LANG=en_US.UTF-8
+    export LC_ALL=en_US.UTF-8
+    export LDFLAGS="-Wl,-headerpad_max_install_names"
+
+    ./configure --prefix="$PREFIX" \
+        --disable-samples \
+        --disable-tests \
```

Comment:
Looks like we are disabling tests. Though maybe we should run enable them so we can verify the binary we built is ok.